tests/databases/molecule/test_get_all.py

```diff
@@ -40,56 +53,43 @@ def test_get_all(mongo_client):
         stk.BuildingBlock('NCCN'),
         stk.BuildingBlock('CCCCC'),
         stk.BuildingBlock('NCCCCN'),
+        stk.BuildingBlock('NCC(CCBr)CCN'),
+        stk.BuildingBlock('NCCNCCC(CCBr)CCN'),
     )
 
-    # Two sets of molecules with two distinct molecules each, and one
-    # shared molecule to test for overlap.
-    molecules1 = (all_molecules[0], all_molecules[1], all_molecules[4])
-    molecules2 = (all_molecules[2], all_molecules[3], all_molecules[4])
+    inchi_molecules = all_molecules[:3]
+    smiles_molecules = all_molecules[3:-2]
```

Comment:
Unsure why you have done it this way - what is the goal here? Why not make it simpler?

Reply:
i feel like this is the simplest way to do it. I am explicitly assigning non-overalapping ranges, this should make it easier to understand the intent of the code and harder to make mistakes in it. for example the index 4 is used twice in the original code and some indices are skipped -> is that a mistake or intentional? hard to say imo

Reply:
Yeah misread the coce above.